frontend/pages/admin/OrgSettingsPage/cards/Advanced/Advanced.tests.tsx

```
@@ -38,24 +38,26 @@ describe("Advanced settings — Activity & data retention", () => {
+      screen.getByLabelText(/Disable hosts online historical reporting/i)
+    ).toBeInTheDocument();
     expect(
       screen.getByLabelText(/Disable vulnerabilities/i)
     ).toBeInTheDocument();
   });
 
   it("starts with both checkboxes unchecked when collection is enabled", () => {
     renderAdvanced();
-    expect(screen.getByLabelText(/Disable hosts online/i)).not.toBeChecked();
-    expect(screen.getByLabelText(/Disable vulnerabilities/i)).not.toBeChecked();
+    expect(screen.getByLabelText(/Disable hosts online historical reporting/i)).not.toBeChecked();
```

ESLint

[error] Replace `screen.getByLabelText(/Disable·hosts·online·historical·reporting/i)` with `⏎······screen.getByLabelText(/Disable·hosts·online·historical·reporting/i)⏎····`

(prettier/prettier)